lib/ops/opsservice/testhelpers.go

```diff
@@ -54,6 +55,8 @@ type TestServices struct {
 	AgentServer rpcserver.Server
 	// Operator is the ops service
 	Operator *Operator
+	// HelmClient is the mock Helm client
```

Comment:
nit: I'd move these either into a file_test.go or mark it with `test` tag.

Reply:
Not following, move what?